cdc/processor/manager.go

```diff
@@ -197,13 +218,22 @@ func (m *Manager) handleCommand() error {
 	case commandTpWriteDebugInfo:
 		w := cmd.payload.(io.Writer)
 		m.writeDebugInfo(w)
+	case commandTpQueryTableCount:
+		count := 0
+		for _, p := range m.processors {
+			count += len(p.GetAllCurrentTables())
```

Comment:
`GetAllCurrentTables` is a method of the `TableExecutor` interface, we aims to make the `TableExecutor` one per table, so the method could be moved to the `Processor` interface if it will be add in the future.

Reply:
Good point, we can do it in a later PR.